compliance/elasticsearch/src/test/java/org/eclipse/rdf4j/sail/elasticsearch/examples/ElasticsearchSailExample.java

```diff
@@ -40,7 +40,11 @@ public class ElasticsearchSailExample {
 	 * @param args
 	 */
 	public static void main(String[] args) throws Exception {
+		long startTime = System.nanoTime();
 		createSimple();
+		long endTime = System.nanoTime();
+        long timeElapsed = endTime - startTime;
+        System.out.println("Execution time in milliseconds for All the Queries: " + timeElapsed / 1000000);
```

Comment:
I don't follow how this addresses the issue? The feature request is about seeing query evaluation time in the RDF4J Workbench tool - a web application. This calculates execution time in a bit of example code, instead.